batch.go

```diff
@@ -0,0 +1,134 @@
+package pgxmock
+
+import (
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgconn"
+)
+
+// BatchElement helps to create batches for testing
+type BatchElement struct {
```

Comment:
Should we rename it to `BatchQuery`? pgx manual uses words `queue` and `query` all along. Should we maybe stick to it?

Sorry for being picky, I have no complaints about functionality, but trying to carefully choose names because it's forever. :-)

Reply:
So true. Thank you very much. I can do it on my own, if you want. Just wanted to confirm we're on the same page

Reply:
I already started writing missing tests for the coverage, so I will do that.

Reply:
Perfect! Thanks!